partial-update-tests/generated/src/main/java/fixtures/bodystring/AutoRestSwaggerBatServiceClientBuilder.java

```diff
@@ -186,7 +186,7 @@ public AutoRestSwaggerBatServiceClientBuilder configuration(Configuration config
 
     /**
      * Sets server parameter.
-     *
+     * 
```

Comment:
This seems like an unintentional space added here. We should revert this to reduce noise in the diff.